mathml/relations/html5-tree/css-inline-style-interface.tentative.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>MathML 'ElementCSSInlineStyle` Mixin Tests</title>
+    <link
+      rel="help"
+      href="https://mathml-refresh.github.io/mathml-core/#dom-and-javascript"
+    />
+    <style>
+        mi { background-color: red;  }
+      }
+    </style>
+    <meta name="assert" content="MathMLElements incorporate a functional ElementCSSInlineStyle interface " />
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+
+  </head>
+  <body tabindex="-1">
```

Comment:
they are hold overs from when I split the original uber test into separate ones, because that one also had to deal with focus-based things. Will remove.